built-ins/authoring-built-ins.md

```diff
@@ -0,0 +1,132 @@
+> ⚠️ Built-in Template Specs are a feature in development and are currently not available for public use. This document describes the initial process required for built-in template spec authoring, but will remain incomplete (contains some placeholders) until the feature has shipped.
+
+
+ **IMPORTANT:** Authoring and contribution of Built-in Template Specs is restricted to **authorized collaborators only** such as internal Microsoft teams and partners. PRs from unauthorized individuals will be rejected with no exceptions.
+
+# Authoring and Contributing Built-in Template Specs
+
+This documents provides general information on what built-in template specs are, and the process involved to contribute new built-in template specs for consumption by a wider population.
+
+## What are built-ins?
+Built-in template specs are [template specs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-specs?tabs=azure-powershell) that deploy cloud components/settings that fulfill scenarios that are seen as useful for a wide variety of customers across Azure. Unlike private template specs which are only available to customers (tenants) who created them, built-in template specs are available to all customers in a specific cloud (such as the Public cloud, or the Chinese national cloud, etc. etc.). Essentially they are "global" template specs that any customer can deploy from any region, and they'll behave in the same way for each customer.
```

Comment:
get rid of `en-us` in the URL

Reply:
I might avoiding calling out Chinese national cloud specifically, and instead say "sovereign clouds" or something more broad.

Reply:
I'll replace all of the en-us references in the docs links being the doc site redirects to the appropriate language by default. For calling out the Chinese national cloud, unless you feel strongly about this I'll leave it as is, because the reference is to a singular cloud rather than something that is plural.